marklogic-data-hub-spark-connector/build.gradle

```diff
@@ -82,22 +82,30 @@ task copyModulesForConnectorFromCoreProject(type: Copy) {
 processResources.dependsOn copyModulesForConnectorFromCoreProject
 
 shadowJar {
-    classifier "all"
+    classifier "pre"
     // Relocate packages of classes that the ML Java Client needs; it can't use the versions provided by Spark
     relocate('okio', 'marklogic.okio')
 
     // Relocating this package from Jackson. Relocating all of com.fasterxml.jackson results in the Scala
```

Comment:
This comment needs to be updated, as it's no longer accurate. I think the entire comment can be removed, in fact.